pandas/tests/indexes/interval/test_interval.py

```diff
@@ -857,6 +858,22 @@ def test_is_all_dates(self):
         year_2017_index = pd.IntervalIndex([year_2017])
         assert not year_2017_index.is_all_dates
 
+    def test_get_value_errors(self):
```

Comment:
Same thing regarding parametrize and the github issue number.

Reply:
changed